page/about-jquery/how-jquery-works.md

```diff
@@ -53,6 +53,8 @@ $( document ).ready(function() {
 });
 ```
 
+<div class="warning">**Note**: The jQuery library exposes its methods and properties via two properties of the <code>window</code> object called <code>jQuery</code> and <code>$</code>. <code>$</code> is simply an alias for <code>jQuery</code> and it's often employed because it's shorter and faster to write.</div>
```

Comment:
Why should this be in a `class="warning"` box? Might as well just have a normal paragraph about this. The warnings always give me the feeling something is a gotcha or something could go wrong, but in this case it's just a concept being introduced.